pkg/openstack/glance.go

```diff
@@ -32,6 +32,9 @@ func ReconcileGlance(ctx context.Context, instance *corev1beta1.OpenStackControl
 		if glance.Spec.DatabaseInstance == "" {
 			glance.Spec.DatabaseInstance = "openstack"
 		}
+		if glance.Spec.StorageClass == "" {
+			glance.Spec.StorageClass = instance.Spec.StorageClass
```

Comment:
This should also help to avoid issues when the `pvc` is created by the `glance-operator` during the bootstrap process